src/components/Book/Book.jsx

```diff
@@ -22,8 +21,8 @@ import CircularProgress from './CircularProgress';
         </div>
         <div className={classes.actionBtn}>
           <button type="button">Comments</button>
-          <button>
-          {/* <button onClick={() => handleRemoveBook(book.item_id)} type="button"> */}
+
+          <button onClick={() => handleRemoveBook(book.item_id)} type="button">
```

Comment:
- [ ] Thank you for your work on this feature! However, as part of the project requirements, it's important that the button for removing a book is implemented as an independent `<Button>` component. Currently, I noticed that you're using the default HTML `<button>` element instead of the required <Button> component so please make sure to update this.
- [ ] This also applies to the button for adding a book in the `BookInput` component, this means the component should accept a function as it's prop so you can reuse it for both adding and removing books. ✔